docs/troubleshooting.md

```diff
@@ -105,9 +105,16 @@ Unable to mount with the following error `daemonize.Run: readFromProcess: sub-pr
 
 Fuse package is not installed. It may throw this error. Run the following command to install fuse <br/> <code> sudo apt-get install fuse </code> for Debian/Ubuntu <br/> <code> sudo yum install fuse </code> for RHEL/CentOs/Rocky <br/>
 
-### ls: reading directory \<mountpath>/\<path>:  Input/output error
+### Encountered unsupported prefixes during listing, or ls: reading directory \<mountpath>/\<path>:  Input/output error
 
-Find out if you have any object(s) with name/prefix having `//` in it or starting with `/`, in the mounted GCS bucket (use `gsutil ls gs://<bucket>/<path>` to find out). If yes, move/rename such objects to name/prefix not having `//` or starting with `/` (e.g. for object/prefix `A//B`, use `gsutil -m mv -r gs://<bucket>/A//* gs://<bucket>/A/`), or delete it (use `gsutil -m rm -r A//`). Refer [semantics](semantics.md#unsupported-object-names) for more details.
+Unable to list unsupported objects in a mounted bucket, i.e. objects which have names with `//` in them or have names starting with `/` e.g. `gs://<bucket>//A` or `gs://<bucket>/A//B` etc. Such objects can be listed by the following command: `gcloud ls -l -r gs://<bucket>/**//**`.
```

Comment:
Cx can see `"Encountered unsupported prefixes during listing: [mdr/logs//]"` type of error. Should we talk about this?